airmozilla/manage/tests/test_archiver.py

```diff
@@ -119,7 +119,8 @@ def mocked_urlopen(request):
 
         sent_email = mail.outbox[-1]
         eq_(sent_email.to, [x[1] for x in settings.ADMINS])
-        ok_('Unable to archive pending event' in sent_email.subject)
+        ok_('Unable to archive processing or pending event'
```

Comment:
Let's just get rid of these and say "Unable to archive event". Which state it is/was in isn't useful.